industry_benchmarks/analysis/README.md

```diff
@@ -23,4 +23,45 @@ The script will automatically find the experimental data for private sets, for p
+| `difference_num_rot_bonds_AB`                   | The difference in the number of rotatable bonds between the ligands.                                                 |
+| `morgan_tanimoto_similarity`                    | The tanimoto similarity of the Morgan fingerprints of the ligands.                                                   |
+| `difference_solvent_accessible_surface_area`    | The difference in the solvent accessible surface area between the ligands.                                           |
+| `atom_pair_dice_similarity`                     | The dice similarity of the atom pair fingerprints.                                                                   |
+| `topological_torsion_dice_similarity`           | The dice similarity of the topological torsion fingerprints.                                                         |
+| `dataset_name`                                  | The name of the dataset.                                                                                             |
+| `partner_id`                                    | The identifier of the industry partner that computed this dataset.                                                   |
+| `exp DDG (kcal/mol)`                            | The experimental DDG in (kcal/mol)                                                                                   |
+| `exp dDDG (kcal/mol)`                           | The experimental uncertainty in (kcal/mol)                                                                           |
+| `solvent/complex_repeat_0/1/2_DG (kcal/mol)`    | The DG calculated for repeat (0/1/2) in phase (solvent/complex).                                                     |
+| `solvent/complex_repeat_0/1/2_dDG (kcal/mol)`   | The error in the calculated DG for repeat (0/1/2) in phase (solvent/complex).                                        |
+| `solvent/complex_repeat_0/1/2_com_drift_max`    | The maximum center of mass drift for the ligand in each repeat (0/1/2) and phase (solvent/complex).                  |
+| `solvent/complex_repeat_0/1/2_ligand_rmsd_max`  | The maximum ligand RMSD in each repeat (0/1/2) and phase (solvent/complex).                                          |
+| `solvent/complex_repeat_0/1/2_smallest_overlap` | The smallest off diagonal element of the mbar overlap matrix for each repeat (0/1/2) and phase (solvent/complex).    |
+| `DG (kcal/mol)`                                 | The MLE calculated DG centered around 0 for the ligand.                                                              |
+| `uncertainty (kcal/mol)`                        | The uncertainty in the DG prediction for the ligand.                                                                 | 
+| `Samples Xns DG`                                | The calculated DG for the transformation using Xns of simulation data.                                               |
```

Comment:
Should this be DDG or is this the DG of the individual legs or MLE DG?

Reply:
Same question for values below

Reply:
This is the DG of the individual legs so one value per phase and repeat, I'll add that to the description.